documentation/docs/guide/GettingStarted/deploy_locally.md

```diff
@@ -18,14 +18,29 @@ cd wordpress/dev
 cp sample-env .env
 ```
 
+::: tip
+you should be using <https://github.com/bcgov/wordpress> as your local WordPress development.
+ensure you pull the latest: `git pull origin main` before you start working on your local WordPress development.
+:::
+
 ## Environment Basics
 
 - Update variables inside `.env` file to reflect your setup.
   - The `CONTENT_DIR` variable is the location of your WordPress content directory.
   - The `TEMP_DIR` variable can be used to import/export db using the wp-cli command.
-    ::: tip
-    When Importing and Exporting databases with the `wp cli` This will be the link between the WordPress container and your local machine.
-    :::
+::: tip
+When Importing and Exporting databases with the `wp cli` This will be the link between the WordPress container and your local machine.
+:::
+::: warning
+If you are using Docker Desktop:
```

Comment:
Although the intended audience probably is Gov, it might be better to say something like (If you want to use rancher desktop)
... if you have docker desktop install then uninstall it
there might be some optimization here, to say as well if you are switching between one or the other uninstall existing and install new